data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/resistance/resistance_positive_value.cfg

```diff
@@ -0,0 +1,31 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [resistance]value=
+##
+# Actions:
+# Give all units 50% resistance to all damage types
+# Attack each other
+##
+# Expected end state:
+# The damage from the attack is reduced by 50%
+#####
+{COMMON_KEEP_A_B_UNIT_TEST "resistance_positive_value" (
+    [event]
+        name = start
+
+        [modify_unit]
+            [filter]
+            [/filter]
+            [effect]
+                apply_to = new_ability
+                [abilities]
+                    {TEST_ABILITY resistance 50 (max_value=100) SELF=yes}
```

Comment:
These two tests are the same: resistance_active_on_both and resistance_positive_value

Reply:
Removed the latter test.